app/assets/v2/css/base.css

```diff
@@ -543,13 +545,25 @@ input[type=text].loading {
 #primary_form {
   max-width: 500px;
   margin: 0 auto;
-  padding: 0 4em;
+  padding: 0 2em;
+}
+
+.submit_bounty .select2-results__option,
+.fulfill_bounty .select2-results__option {
+  font-size: 14px;
```

Comment:
How about we keep it in `rem` instead of `px`? I think 0.9rem looks okay. Can you verify? Else on changing the base font-size, this font-size still remains 14px, which is kind of an unwanted scenario, I guess.

Reply:
Btw It's ~0.85rem